dotnet/src/Functions/Functions.Prompty/Extensions/PromptyKernelExtensions.cs

```diff
@@ -20,21 +20,39 @@ public static class PromptyKernelExtensions
     /// Create a <see cref="KernelFunction"/> from a prompty file.
     /// </summary>
     /// <param name="kernel">kernel</param>
-    /// <param name="promptyPath">path to prompty file.</param>
+    /// <param name="promptyFilePath">Prompty template.</param>
     /// <param name="promptTemplateFactory">prompty template factory, if not provided, a <see cref="LiquidPromptTemplateFactory"/> will be used.</param>
     /// <param name="loggerFactory">logger factory</param>
     /// <returns><see cref="KernelFunction"/></returns>
     /// <exception cref="ArgumentNullException"></exception>
     /// <exception cref="NotSupportedException"></exception>
```

Comment:
It'd be nice to clean up the XML comments, in general. I can do it after if you don't want to.